docs/README.md

```diff
@@ -1,91 +1,70 @@
+vtex add vtex.on-view@1.x
+```
 
-### `block-1` props
+## API
 
-| Prop name    | Type            | Description    | Default value                                                                                                                               |
-| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
-| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+### `useOnView`
 
+It will call the `onView` function when the HTML element is inside the viewport. Differs from [react-intersection-observer](https://www.npmjs.com/package/react-intersection-observer) in that this hook doesn't use setState, using a callback approach instead, to improve performance by preventing re-rendering.
 
-### `block-2` props
+```tsx
+import React from 'react'
+import { useOnView } from 'vtex.on-view'
 
-| Prop name    | Type            | Description    | Default value                                                                                                                               |
-| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
-| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+function MyComponent() {
+  const element = useRef<HTMLDivElement | null>(null)
 
-Prop types are: 
+  useOnView({
+    ref: element,
+    once: true,
+    onView: () => {
+      console.log('Element is visible')
+    },
+  })
 
-- `string` 
-- `enum` 
-- `number` 
-- `boolean` 
-- `object` 
-- `array` 
+  return <div ref={element}>Hello World</div>
+}
 
-When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:
+export default MyComponent
+```
 
-- `propName` object:
+### `useRenderOnView`
 
-| Prop name    | Type            | Description    | Default value                                                                                                                               |
-| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
-| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+It renders a dummy element while the element is not inside the viewport.
 
+```tsx
+import React from 'react'
+import { useRenderOnView } from 'vtex.on-view'
 
-Remember to also use this Configuration section to  **showcase any necessary disclaimer** related to the app and its blocks, such as the different behavior it may display during its configuration. 
+function MyComponent() {
+  const element = useRef<HTMLDivElement | null>(null)
 
-## Modus Operandi *(not mandatory)*
+  const { hasBeenViewed, dummyElement } = useRenderOnView({
```

Comment:
Can we replace the `dummy` with something less ableist without being a breaking change 😢 ?

Reply:
(https://developers.google.com/style/inclusive-documentation#ableist-language)

Reply:
Great! Changed!